tests/data/example_config.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "test_run",
+    "user_id": "test_user1@mail.com",
+    "job_id": "BaxDa",
+    "baseurl": "http://localhost:3000",
+    "baseurl_v2": "http://localhost:3000",
+    "job_dir": "./tests/jobs/BaxDa",
+    "cdate": 1649767324961,
+    "mdate": 1649767324961,
+    "match_group": [
+        "ABC.cc"
+    ],
+    "dataset": {
+        "directory": "./tests/jobs/BaxDa"
+    },
+    "model": "specter+mfr",
+    "paper_id": "KHnr1r7h",
+    "model_params": {
+        "use_title": false,
+        "use_abstract": true,
+        "average_score": true,
+        "max_score": false,
+        "skip_specter": false,
+        "specter_batch_size": 16,
+        "mfr_batch_size": 50,
+        "use_cuda": false,
+        "name": "specter+mfr",
+        "work_dir": "./tests/jobs/BaxDa",
```

Comment:
same here. I would avoid all these directories here. They should be part of the server configuration.